### PR TITLE
Added smrt link 11 fields to configuration.

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
 # Traction service
 
-[![Build Status](https://travis-ci.org/sanger/traction-service.svg?branch=devel)](https://travis-ci.org/sanger/traction-service)
-
-Rails app which exposes a RESTful API.
+Rails app which exposes a RESTful API for a Long Read LIMS
+
+- Add details aboout traction ui
```
Comment (sdjmchattie) [inline README.md:5]:
```suggestion
- Add details about traction ui
```

File: app/models/pacbio/well.rb [hunk 2]
```diff
@@ -9,13 +9,8 @@ class Well < ApplicationRecord
...
     include SampleSheet::Well
 
-    # We should get rid of the below enum,
-    # as we are not supporting sequencing_mode in SMRTLink v10
-    # but if we keep the field in the database, for auditing,
-    # then we need this to translate the integer to a value
-    enum sequencing_mode: { 'CLR' => 0, 'CCS' => 1 }
-
-    enum generate_hifi: { 'In SMRT Link' => 0, 'On Instrument' => 1, 'Do Not Generate' => 2 }
+    # Do not delete until the column has been migrated
```
Comment (sdjmchattie) [inline app/models/pacbio/well.rb:12]: Is there a ticket that can be referenced here for that migration?  I can see this staying here for the long term without something referencing when it is removable.

File: app/pacbio/smrt_link.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# tried to namespace this under module Pacbio
+# not playing ball
+# moduel SmrtLink
```
Comment (sdjmchattie) [inline app/pacbio/smrt_link.rb:5]:
```suggestion
```

**Review by sdjmchattie — APPROVED** [review]

File: app/models/pacbio/well.rb (hunk 2 above)
Comment (stevieing) [reply app/models/pacbio/well.rb:12]: I can remove it as it is referenced in the migration

**Pull request merged**